.header.md

```diff
@@ -0,0 +1,80 @@
+# AWS VPC Module
+
+This module can be used to deploy a pragmatic VPC with various subnets types in # AZs. Common deployment examples can be found in [examples/](./examples/). Subnet CIDRs can be explicitly set via list of string argument `cidrs` or set via a number `netmask` argument.
```

Comment:
The examples relative link will be broken in the tf-registry. Recommend using the absolute link (https://github.com/aws-ia/terraform-aws-vpc/tree/main/examples) to achieve the desired results.